parts/plugins/apache.py

```diff
@@ -103,8 +103,8 @@ def __init__(self, name, options):
         super().__init__(name, options)
 
         self.build_packages.extend(['libapr1-dev', 'libaprutil1-dev',
-                                    'libpcre3-dev'])
-        self.stage_packages.extend(['libapr1', 'libaprutil1', 'libpcre3'])
+                                    'libpcre3-dev', 'libssl-dev', 'zlib1g-dev'])
```

Comment:
That's the thing, how to know for sure? Especially if building a 32bit image from a 64bit system. Won't there be any surprises?